14.5/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# vim:set ft=dockerfile:
+
+FROM cimg/base:2020.06
```

Comment:
Might want to switch these to 2020.07? Maybe 20.04 too

Reply:
There's nothing significant that we need to pull in from the July snapshot. As for Ubuntu 20.04, our images won't switch over to it until August or September.